src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -91,6 +98,84 @@ public void changesTransactionIsolationLevel() throws Exception {
     }
 
     /**
+     * JdbcSession can run a function (stored procedure) with
+     * output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithOutParam() throws Exception {
+        final DataSource dsrc = JdbcSessionITCase.source();
+        new JdbcSession(dsrc).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS users (name VARCHAR(50))"
+        ).execute().sql("INSERT INTO users (name) VALUES (?)")
+        .set("Jeff Charles").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION getUser(username OUT text)",
+                " AS $$ BEGIN SELECT name INTO username from  users; END;",
+                " $$ LANGUAGE plpgsql;"
+            )
+        ).execute().commit();
+        final Object[] result = new JdbcSession(dsrc)
+            .sql("{call getUser(?)}")
+            .prepare(
+                new Preparation() {
+                    @Override
+                    public void prepare(
+                        final PreparedStatement stmt) throws SQLException {
+                        ((CallableStatement) stmt)
+                            .registerOutParameter(1, Types.VARCHAR);
+                    }
+                }
+             )
+            .call(new StoredProcedureOutcome<Object[]>(1));
+        MatcherAssert.assertThat(result.length, Matchers.is(1));
+        MatcherAssert.assertThat(
+            result[0].toString(),
+            Matchers.containsString("Charles")
+        );
+    }
+
+    /**
+     * JdbcSession can run a function (stored procedure) with
+     * input and output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithInOutParam() throws Exception {
+        final DataSource dsrc = JdbcSessionITCase.source();
+        new JdbcSession(dsrc).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS usersid (id INTEGER, name VARCHAR(50))"
+        ).execute().sql("INSERT INTO usersid (id, name) VALUES (?, ?)")
+        .set(1).set("Marco Polo").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION getUserById(uid IN INTEGER,",
+                " usrnm OUT text) AS $$ BEGIN",
```

Comment:
@mkordas To be honest, I'm not sure. It's the first time I work with PSQL and it's the same in all (I mean ALL) the stored procedure examples I could find. Search for "postgresql $$ operator" doesn't return anything either. Do you know it works without it? Even if it works, I assume there must be a reason for it. It's also there in the official examples [here](https://www.postgresql.org/docs/8.0/static/plpgsql.html)